### DISCUSSION TITLE
Scoop package

### DISCUSSION THREAD

**Discussion by trymeouteh** [DevToys-app/DevToys#1278] [Ideas]
Please release a Windows scoop package

**Comment by EphemeralDev** [comment]
There does currently exist a [scoop manifest](https://github.com/ScoopInstaller/Nonportable/blob/master/bucket/devtoys-np.json), but it is for the nonportable 1.0 release. The 2.x releases are still pre-release but target a much wider variety of installation options. As such, the existing non portable manifest should probably be updated to reflect the new 2.x releases. In addition I think there should be new manifests created for main (cli) and extras (gui) [buckets](https://github.com/ScoopInstaller/Scoop?tab=readme-ov-file#known-application-buckets).
